users/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from users.managers import UserManager
+
+
+class User(AbstractUser):
+    username = None
```

Comment:
why?

Reply:
Without this setting I was getting programming error ('column users_user.username does not exist'). Built-in Django User model expects username field which is required.